dist/stat_data.py

```diff
@@ -565,6 +565,11 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_checkpoint_fsync_post_duration', 'transaction fsync duration for checkpoint after allocating the transaction ID (usecs)', 'no_clear,no_scale'),
     TxnStat('txn_hs_ckpt_duration', 'transaction checkpoint history store file duration (usecs)'),
     TxnStat('txn_checkpoint_generation', 'transaction checkpoint generation', 'no_clear,no_scale'),
+    TxnStat('txn_checkpoint_prep_running', 'transaction checkpoint prepare currently running', 'no_clear,no_scale'),
```

Comment:
Alphabetical ordering nit `txn_checkpoint_prep_running` should go after `txn_checkpoint_prep_recent`

Reply:
Thanks for catching that! Fixed.